functests/3_performance_status/status.go

```diff
@@ -103,6 +103,21 @@ var _ = Describe("Status testing of performance profile", func() {
 
 			mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
 		})
+
+		It("[test_id:40402] Tuned profile status tied to Performance Profile", func() {
+			// Creating bad Tuned object that leads to degraded state
+			badTuned := createBadTuned("openshift-cause-tuned-failure", "openshift-cluster-node-tuning-operator")
+			err = testclient.Client.Create(context.TODO(), badTuned)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Waiting for performance profile condition to be Degraded")
+			profiles.WaitForCondition(testutils.NodeSelectorLabels, v1.ConditionDegraded, corev1.ConditionTrue)
```

Comment:
if this fails, is the bad tuned removed anyway?

Reply:
No. Do you want to remove it anyway in order to leave the system in a clean state after the test is falling?

Reply:
Yes please. We should always do all we can to leave the cluster in a clean/consistent state.

Reply:
So the same problem apply in [mcp]( https://github.com/openshift-kni/performance-addon-operators/blob/747cbc5b937af9aded6afda395c8359c747db471/functests/3_performance_status/status.go#L94) as well

Reply:
right. We can just add a TODO here and fix on a separate PR.